client/dashboards.go

```diff
@@ -0,0 +1,71 @@
+package client
```

Comment:
It may be better to change the filename based on the resource name change - `data_ui_views.go`?